content/de/includes/user-guide-content-moved.md

```diff
@@ -0,0 +1,3 @@
+The topics in the [User Guide](/docs/user-guide/) section of the Kubernetes docs
```

Comment:
It is better to not have English content present as it may change while you're translating.